{
  doTest(true,PyBundle.message("refactoring.make.function.top.level.error.special.usage.of.self"));
}
